### PR TITLE
Rappel affaires course V1

### PR DESCRIPTION
**Pull request by jeromelebaron**

### CODE REVIEW

File: scripts/Rappel_affaire_de_course.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  newReminder.title = 'Préparer les affaires de course';
+  newReminder.calendar = reminderCalendar;
+  newReminder.dueDate = createMorningDateDayBeforeFromCalendarEvent(calendarEvent);
+  newReminder.save();
+}
+
+function createMorningDateDayBeforeFromCalendarEvent(calendarEvent) {
+  let startDate = calendarEvent.startDate;
+  return new Date(startDate.getFullYear(), startDate.getMonth(), startDate.getDate() - 1, 7, 30, 0, 0);
+}
```
Comment (mzottola) [inline scripts/Rappel_affaire_de_course.js]: carriage return missing

File: scripts/Rappel_affaire_de_course.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+let weekRunningCalendarEvents = await CalendarEvent.thisWeek([runningCalendar]);
+let efWeekRunningCalendarEvents = extractEfWeekRunningCalendarEvents(weekRunningCalendarEvents);
+
+const rappelReminderName = 'Rappels';
+let rappelCalendar = await Calendar.forRemindersByTitle(rappelReminderName);
+efWeekRunningCalendarEvents.forEach(event => createReminder(event, rappelCalendar));
+
+Script.complete();
+
+function extractEfWeekRunningCalendarEvents(runningCalendarEvents) {
```
Comment (mzottola) [inline scripts/Rappel_affaire_de_course.js:17]: What `Ef` means ?

File: scripts/Rappel_affaire_de_course.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  return runningCalendarEvents
+  .filter(event => isFuture(event))
```
Comment (mzottola) [inline scripts/Rappel_affaire_de_course.js]: No method reference in javascript ? :scream:

File: scripts/Rappel_affaire_de_course.js (hunk 1 above)
Comment (jeromelebaron) [reply scripts/Rappel_affaire_de_course.js]: Done

**Pull request merged**